client/packages/system/src/Item/Components/StockItemSearchInput/StockItemSearchInput.tsx

```diff
@@ -50,11 +65,23 @@ export const StockItemSearchInput: FC<StockItemSearchInputProps> = ({
   const formatNumber = useFormatNumber();
   const selectControl = useToggle();
 
-  const debounceOnFilter = useDebounceCallback(
-    (searchText: string) => onFilter(searchText),
-    [onFilter],
-    DEBOUNCE_TIMEOUT
-  );
+  const filterByNameAndCode = (
+    options: ItemStockOnHandFragment[],
+    state: FilterOptionsState<ItemStockOnHandFragment>
+  ) =>
+    options.filter(option => {
+      const searchValue = RegexUtils.escapeChars(state.inputValue);
+      return (
+        RegexUtils.includes(searchValue, option.code) ||
+        RegexUtils.includes(
+          getItemNameFilterValue(searchValue, selectedCode),
+          option.name
+        )
+      );
+    });
+
+  const getOptionLabel = (option: ItemStockOnHandFragment) =>
+    `${option.code} ${option.name}`;
```

Comment:
This is also used in quite a few item search components, can this be moved to utils to use across?

Reply:
Moved out of the component [f1be7e1](https://github.com/msupply-foundation/open-msupply/pull/6228/commits/f1be7e19e087789a780d5af81ea17883804ea225) - found that the other item search components are not used, so actually just removed them [73743bc](https://github.com/msupply-foundation/open-msupply/pull/6228/commits/73743bca1fddd9541cf68315982839034cf3311e) 🎉